test/suite/test_compact07.py

```diff
@@ -104,24 +104,24 @@ def test_compact07(self):
         self.session.checkpoint()
 
         # There should be more free space in the last created tables compared to the very first one.
+        min_free_space = 0
         for i in range(self.n_tables):
-            self.assertGreater(self.get_free_space(uri), free_space_20)
+            free_space = self.get_free_space(uri)
+            self.assertGreater(free_space, free_space_20)
+            if i == 0:
+                min_free_space = free_space
+            elif free_space < min_free_space:
+                min_free_space = free_space
 
         # Enable background compaction with a threshold big enough so it does not process the first
         # table created but only the others with more empty space.
-        self.session.compact(None, f'background=true,free_space_target={free_space_20 + 1}MB')
-
-        # Wait for the background server to wake up.
-        compact_running = self.get_bg_compaction_running()
-        while not compact_running:
-            time.sleep(1)
-            compact_running = self.get_bg_compaction_running()
-        self.assertEqual(compact_running, 1)
+        assert min_free_space - 1 > free_space_20
```

Comment:
Should we use `self.assertGreater` here rather than a plain `assert`?

Reply:
Sure b5259ec22c73c06c326b3ef782841eb2980378f4